hircluster.c

```diff
@@ -4146,7 +4171,6 @@ int redisClusterAsyncFormattedCommand(redisClusterAsyncContext *acc,
 
     cad = cluster_async_data_get();
     if (cad == NULL) {
-        __redisClusterAsyncSetError(acc, REDIS_ERR_OOM, "Out of memory");
         goto error;
```

Comment:
Will change this back to OOM, cluster_async_data_get() return NULL when its malloc() fails
`goto error;`  will change to `goto oom;`